src/libs/actions/connections/index.ts

```diff
@@ -0,0 +1,112 @@
+import Onyx from 'react-native-onyx';
+import type {OnyxUpdate} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import * as API from '@libs/API';
+import {WRITE_COMMANDS} from '@libs/API/types';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PolicyConnectionName} from '@src/types/onyx/Policy';
+
+function removePolicyConnection(policyID: string, connectionName: PolicyConnectionName) {
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                connections: {
+                    [connectionName]: null,
+                },
+            },
+        },
+        {
+            onyxMethod: Onyx.METHOD.SET,
+            key: `${ONYXKEYS.COLLECTION.POLICY_CONNECTION_SYNC_PROGRESS}${policyID}`,
+            value: null,
+        },
+    ];
+    const failureData: OnyxUpdate[] = [
+        // {
+        //     onyxMethod: Onyx.METHOD.MERGE,
+        //     key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+        //     value: {
+        //         errorFields: {
+        //             avatar: ErrorUtils.getMicroSecondOnyxError('avatarWithImagePicker.deleteWorkspaceError'),
+        //         },
+        //     },
+        // },
+    ];
```

Comment:
Please ignore this, I will address in a follow up PR. I want to avoid delaying this more and getting conflicts.

Reply:
hehe, no I meant that I didn't want to spend time on fully implementing `RemoveConnection` since to my understanding it is out of the scope of this PR. I just added it because it makes testing easier.

I didn't want to keep increasing the scope and making the review process longer, this is what causes conflicts.

Reply:
Having said that, I don't know if we have a separate issue for it, I'll create one.

@hayata-suenaga do we have an issue for disconnecting the policy?